pkg/controller/common/beat/otherbeat/otherbeat.go

```diff
@@ -0,0 +1,33 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package otherbeat
+
+import (
+	commonbeat "github.com/elastic/cloud-on-k8s/pkg/controller/common/beat"
+	"github.com/elastic/cloud-on-k8s/pkg/utils/pointer"
+)
+
+const (
+	Type commonbeat.Type = "otherbeat"
```

Comment:
Unused? Also I think we should not surface the name `otherbeat` anywhere to the user (but I think the name is OK for internal use)

Reply:
Yes, unused. I'll remove for now, we can reintroduce it when there is a need.